packages/core/helper-plugin/lib/src/components/FilterListURLQuery/AttributeTag.js

```diff
@@ -42,6 +42,18 @@ const AttributeTag = ({ attribute, filter, onClick, operator, value }) => {
     formattedValue = formatNumber(value);
   }
 
+  // Handle custom input
+  if (attribute.metadatas.customInput) {
```

Comment:
Could you please add tests for this new case? :) From the code is is not very obvious what this does ...